### PR TITLE
Add warning message on validateRequest

### PR DESCRIPTION
**Pull request by edolix**

### CODE REVIEW

File: lib/webhooks/webhooks.js [hunk 1]
```diff
@@ -124,8 +124,20 @@ function validateRequest(authToken, twilioHeader, url, params) {
   const signatureWithoutPort = getExpectedTwilioSignature(authToken, urlWithoutPort, params);
   const validSignatureWithPort = scmp(Buffer.from(twilioHeader), Buffer.from(signatureWithPort));
   const validSignatureWithoutPort = scmp(Buffer.from(twilioHeader), Buffer.from(signatureWithoutPort));
+  const isValid = validSignatureWithoutPort || validSignatureWithPort
+  
+  if (!isValid) {
+    const searchParams = new URLSearchParams(urlObject.query)
+    const hasProjectId = searchParams.has('project_id')
+    const hasCallId = searchParams.has('call_id')
+    const hasMessageId = searchParams.has('message_id')
```
Comment (edolix) [inline lib/webhooks/webhooks.js:133]: @danieleds not sure if these cases are valid but i'm checking the query string too. Thoughts?

File: lib/webhooks/webhooks.js [hunk 2]
```diff
@@ -124,8 +124,20 @@ function validateRequest(authToken, twilioHeader, url, params) {
...
+    const { project_id, call_id, call, message_id } = params || {}
+    if (hasProjectId || hasCallId || hasMessageId || project_id || call_id || message_id || call?.project_id) {
+      console.warn('It seems that you are trying to validate a RELAY request using the Compatibility SDK "validateRequest" method. Please use method XXX from package YYY to validate all types of SignalWire requests (RELAY and Compatibility).')
```
Comment (edolix) [inline lib/webhooks/webhooks.js:136]: Note to replace `XXX` and `YYY`

File: lib/webhooks/webhooks.js (hunk 1 above)
Comment (danieleds) [reply lib/webhooks/webhooks.js:133]: @edolix I may be wrong but I don't think we ever use query strings for RELAY requests. Anyway having it won't hurt

File: lib/webhooks/webhooks.js (hunk 2 above)
Comment (edolix) [reply lib/webhooks/webhooks.js]: 5d832b7cb57eddaec99daead96fef55ae0b1701d

**Review by iAmmar7 — APPROVED** [review]

**Review by danieleds — APPROVED** [review]

**Pull request merged**